protected-api-access-protected-api/app.js

```diff
@@ -0,0 +1,88 @@
+const msal = require("@azure/msal-node");
+const express = require("express");
+const https = require("https");
+const jsonwebtoken = require("jsonwebtoken");
+const jwksRsa = require("jwks-rsa");
+
+const SERVER_PORT= 8080;
+
+const tenant = '**TENANT_GOES_HERE**';
+
+const config =  {
+        auth:   {
+                clientId: '**CLIENT_ID_GOES_HERE**',
+                clientSecret: '**CLIENT_SECRET_GOES_HERE**',
+                authority: `https://login.microsoftonline.com/${tenant}`
+        }
+};
+
+const cca = new msal.ConfidentialClientApplication(config);
+
+const app = express();
+
+const validateJwt = (req, res, next) => {
+    const authHeader = req.headers.authorization;
+
+    if (authHeader) {
+        const token = authHeader.split(' ')[1];
+
+        const validationOptions = {
+            audience: config.auth.clientId,
+            issuer: config.auth.authority + '/v2.0'
+        }
+
+        jsonwebtoken.verify(token, getSigningKeys, validationOptions, (err, payload) => {
+            if (err) {
+                console.log(err);
+                return res.sendStatus(403);
+            }
+           next();
+        });
+    }
+};
+
+
+const getSigningKeys = (header, callback) => {
+    var client = jwksRsa({
```

Comment:
Always name things explicity, it helps when stuff is brough into docs.  So for example, this would be `jwkRsaClient` or `jwkRsaValidator` or something

Reply:
Noted. Making suggested change.

Reply:
+1